hnn_core/network.py

```diff
@@ -57,6 +57,87 @@ def read_spikes(fname, gid_dict=None):
     return Spikes(times=spike_times, gids=spike_gids, types=spike_types)
 
 
+def _create_coords(n_pyr_x, n_pyr_y, n_common_feeds, p_unique_keys,
+                   zdiff=1307.4):
+    """Creates coordinate grid.
+
+    Parameters
+    ----------
+    n_pyr_x : int
+        The number of Pyramidal cells in x direction.
+    n_pyr_y : int
+        The number of Pyramidal cells in y direction.
+    n_common_feeds : int
+        The number of common feeds.
+    p_unique_keys : list of str
+        The keys of the dictionary p_unique.
```

Comment:
So far we have operated under the assumption that there is only one type of common feed: rhythmic. For the sake of consistency, we may want to treat common feeds the same way we treat unique keys, with the exception that common feeds will always be assigned to a position at the origin.

Reply:
humm ... okay so how does this relate to the PR here? sorry I am a bit lost. Note unique feeds are also assigned to a position in origin.

Reply:
Right now, we enter common feeds by supplying `n_common_feeds`. There is nothing intrinsically wrong with this, however, I think it would be clearer if instead we supplied the argument `p_common_keys` so that the pos_dict items associated with common feeds have keys that specify the _type_ of common feed (e.g., tonic, rhythmic, etc.)

Reply:
@jasmainak what do you think about using an input arg similar to `p_unique_keys` except for the common feeds?

Reply:
sorry I had missed that previous comment 3 days ago. Actually I like that the common feed has a single key because the position will be the same no matter what the spike pattern of the feed is like. But I agree, there is an inconsistency between unique and common. We should try to fix this everywhere in the codebase as part of a single PR.

what's your opinion on the plot? should I remove the feeds? or how should I fix it?

Reply:
I guess I don't understand why unique feeds can't then be input the same way (i.e., with just a number to specify the amount of unique feeds) since they are all assigned to the same location anyway.

Reply:
>what's your opinion on the plot? should I remove the feeds? or how should I fix it?

See [below](https://github.com/jonescompneurolab/hnn-core/pull/122#issuecomment-664476075) for my thoughts on the plot.

Reply:
We can probably change it, but we need to change it deep inside the code as in here: https://github.com/jonescompneurolab/hnn-core/blob/master/hnn_core/pyramidal.py#L948. The fact that class attributes are used instead of explicitly passed arguments makes the code very tangled and inter-connected :( I opened an issue #123 to investigate this separately